yetanotherco/aligned_layer
# feat (batcher): send batches sequentially

Batcher does not finalize batch if previous one is not finished being posted

# To test

1. Start anvil, batcher, aggregator and operator (make sure to start anvil with block time)
2. Send lots of proofs (useful for this `while sleep 1; do make batcher_send_plonk_bn254_burst &; done`)
3. Batcher should not get nonce too low ethereum error you should see log 
```
 Batch is currently being posted. Waiting for the current batch to be finalized...
```
